transform/app/settings.py

```diff
@@ -108,8 +132,45 @@ def __init__(self, **kwargs):
                 "SEND_TO_SOLR or/and SEND_TO_S3 needs to be set to True. Otherwise, program will not have any effect"
             )
 
+        if self.CREATE_RELATIONS:
+            self.RELATIONS = self.get_relations_config()
+
         self.COLLECTIONS = self.get_collections_config()
 
+    def get_relations_config(self) -> dict:
+        """Get relations config"""
+
+        relations = {
+            self.TMP_DIRECTORY: {
```

Comment:
In process_data file, I have included this instead of in settings.
```
combined_pq = os.path.join(settings.TMP_DIRECTORY, "combined_data.parquet")
organization_pq = os.path.join(settings.TMP_DIRECTORY, "organization.parquet")
project_pq = os.path.join(settings.TMP_DIRECTORY, "project.parquet")
```